docs/examples/python-requests-jwt.md

```diff
@@ -0,0 +1,49 @@
+## Accessing PostgREST API using Python Requests and Requests-JWT
+
+This code relies on setting up the PostgreSQL auth functions and grants correctly first. Follow these instructions:
+
+http://postgrest.com/examples/users/
+
+After completing the configuration, be sure to create a user with email, password, role, and verified flag. We'll use that user to login in the code below.
+
+Install the required libraries
+
+    pip install requests
+    pip install requests-jwt
+
+Then, from a python interpreter or script:
+
+    import requests
+    import requests_jwt
+    import json
+
+    # email and password auth through postgrest
+    resp = requests.post('http://localhost:3000/rpc/login', json={"email": "you@yours.com", "pass": "dog"})
+    if resp.status_code != 200:
+        raise Exception()
+
+    # JWT auth using the token above
+    token = json.loads(resp.text)['token']
+    auth = requests_jwt.JWTAuth(token)
+    if r.status_code != 200:
+        raise Exception()
+
+    # Make a request to get the range header
+    r = requests.get('http://localhost:3000/posts', auth=auth)
```

Comment:
This might not be a good idea since it would request the entire list of posts which could be big. You can actually request a limited range initially (like 0-9) and the response Content-Range header will still give you the total for all posts.